Apps/W1/EDocumentConnectors/Continia/app/src/Implementation/ConnectionSetup.Table.al

```diff
@@ -0,0 +1,193 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Continia;
+
+table 6390 "Connection Setup"
+{
+    Access = Internal;
+    DataClassification = CustomerContent;
+    DataPerCompany = false;
+    Extensible = false;
+
+    fields
+    {
+        field(1; PK; Code[10])
+        {
+            Caption = 'Primary Key';
+        }
+        field(3; "Client Id"; Guid)
+        {
+            Caption = 'Client Id';
+            DataClassification = EndUserIdentifiableInformation;
+        }
+        field(4; "Client Secret"; Guid)
+        {
+            Caption = 'Client Secret';
+            DataClassification = EndUserIdentifiableInformation;
+        }
+        field(5; "Local Client Identifier"; Code[50])
+        {
+            Caption = 'Local Client Identifier';
+            DataClassification = EndUserIdentifiableInformation;
+            Editable = false;
+        }
+        field(6; "Access Token"; Guid)
+        {
+            Caption = 'Access Token';
+        }
+        field(7; "Token Timestamp"; DateTime)
+        {
+            Caption = 'Token Timestamp';
+        }
+        field(8; "Expires In (ms)"; Integer)
+        {
+            Caption = 'Expires In (ms)';
+        }
+        field(9; "Subscription Status"; Enum "Subscription Status")
+        {
+            Caption = 'Subscription Status';
+            ToolTip = 'Specifies the status of the Continia subscription.';
+        }
+        field(10; "No. Of Participations"; Integer)
+        {
+            CalcFormula = count(Participation);
```

Comment:
The ConnectionSetup tables has `DataPerCompany = false;`, but the Participation is `DataPerCompany = true;` (Default-Value).

Is this FlowField then really useful or should the ToolTip at least mention that the information is only for the current company.

If you change the DataCompany on this table then keep in mind that you should update the DataScope for the IsolatedStorage, too.